types/pg/index.d.ts

```diff
@@ -183,6 +183,9 @@ export class Pool extends events.EventEmitter {
     readonly idleCount: number;
     readonly waitingCount: number;
 
+    readonly ending: boolean;
+    readonly ended: boolean;
```

Comment:
These aren't strictly readonly in node-postgres--they're not getters--but I think the intention is that callers shouldn't be modifying them.